### PR TITLE
🖌️ Style more type names

### PR DESCRIPTION
**Pull request by Qrtn**
## Description

Fixes #<number>

### CODE REVIEW

File: src/components/form/LocationForm.js [hunk 1]
```diff
@@ -91,6 +92,9 @@ const LocationStep = ({ typeOptions }) => (
       isMulti
       closeMenuOnSelect={false}
       blurInputOnSelect={false}
+      formatOptionLabel={(option) => (
+        <TypeName typeId={option.value} isTwoLines />
```
Comment (SirajChokshi) [inline src/components/form/LocationForm.js:96]: Should we call this `hasLineBreak`

File: src/components/ui/TypeName.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+
+import { useSearch } from '../../contexts/SearchContext'
+import { useSettings } from '../../contexts/SettingsContext'
+
+const CommonName = styled.span`
+  display: ${({ isTwoLines }) => (isTwoLines ? 'block' : 'inline')};
+
+  font-weight: bold;
+  font-size: 14px;
```
Comment (SirajChokshi) [inline src/components/ui/TypeName.js:10]: use rems (`font_size / 16`), so here it would be

File: src/contexts/SettingsContext.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { createContext, useContext, useReducer } from 'react'
  */
 const DEFAULT_SETTINGS = {
   showLabels: false,
-  showScientificNames: false,
+  showScientificNames: true,
```
Comment (SirajChokshi) [inline src/contexts/SettingsContext.js:10]: 😛

**Review by SirajChokshi — APPROVED** [review]
That _was_  a tasty PR 😋

73aeee4a9b9bf3f28e02624a0e9d3c3f

**Comment by Qrtn** [comment]
Closed because incorrect branch name.

**Pull request closed without merging**